anacron/log.c

```diff
@@ -79,7 +79,7 @@ make_msg(const char *fmt, va_list args)
     /* There's some confusion in the documentation about what vsnprintf
      * returns when the buffer overflows.  Hmmm... */
     len = vsnprintf(msg, sizeof(msg), fmt, args);
-    if (len >= sizeof(msg) - 1)
+    if (0 <= len && (size_t) len >= sizeof(msg) - 1)
```

Comment:
Can you please change it to rather cast the right side to int? And not doing the explicit comparison to zero.

Reply:
Or handle also the negative len as separate case by strcpying some message like "(vnsprintf error)" into msg.

Reply:
These `0 <= len` are now done better way in commit 475e869ef303e1eca2d2951253b61b6d03b2395c.

Pull request is also rebased on top of more recent origin/master that fixed a merge conflict.